openqa-label-known-issues

```diff
@@ -134,14 +134,15 @@ investigate_issue() {
     # against even in case when autoinst-log.txt is missing the details, e.g.
     # see https://progress.opensuse.org/issues/69178
     echo "$reason" >> "$out"
-    if [[ "$curl_output" != "200" ]] && [[ "$curl_output" != "301" ]]; then
+    # Control flow when there is a curl issue but still want to process job due to reason var
+    # even if autoinst-log.txt not found but assume is missing.
```

Comment:
I don't understand that part of the sentence

Reply:
I'm also not sure how to interpret the sentence. It isn't grammatically correct.

I would also avoid using the expression "Control flow when …" because it is very clunky. Just start with a common "handle the case when …" or maybe you can come up with something less generic.

Reply:
@Martchus I changed that. Also I reply here https://github.com/os-autoinst/scripts/pull/342#discussion_r1733998927